### ISSUE TITLE
Node-RED npm is missing locales

### ISSUE THREAD

**Issue by Brezensalzer** [SensorsIot/IOTstack#60]
Hello,
in my Node-RED flow I have a javascript statement that puts out a datetime string to the dashboard with "var ts = d.toLocaleString('de-DE');".
So I expect to get the time like "13:27:14". But in the dashboard I see "1:27:14 PM" which is the locale "en-US".
Probably that happens because the underlying npm was built with "small-icu" and not with "full-icu"?

**Comment by Slyke** [comment]
Yep, I think you're right. I will investigate this and see if it's possible to easily add.
